doc/apiary/v2/fiware-ngsiv2-rc-2016_05_31.apib

```diff
@@ -1848,188 +1779,6 @@ Response:
 
 + Response 204
 
-# Group Registrations
-
```

Comment:
I would leave here a note indicating that this is a group of operations not part of this release but ongoing work.